### ISSUE TITLE
Running multiple cards in parallel is slower(nearly twice) than a single card

### ISSUE THREAD

**Issue by Amanda-Barbara** [vllm-project/vllm#935]
Hi, when I am running four A100 with parameter tensor_parallel_size is 4 in parallel, I found that the speed  is slower(nearly twice) than a single card. can you explain what causes this and how to solve it. Thank you.

**Comment by dutsc** [comment]
I use 3090 and had the same problem as you.

**Comment by hmellor** [comment]
Closing as duplicate of #689
